.snippets/code/tutorials/eth-api/foundry-start-to-end/terminal/test.md

```diff
@@ -0,0 +1,16 @@
+<div id="termynal" data-termynal>
+    <span data-ty="input">foundry % forge test</span>
+    <span data-ty>[⠊] Compiling...</span>
+    <span data-ty>No files changed, compilation skipped</span>
+    <br>
+    <span data-ty>Ran 1 test for test/MyToken.t.sol:MyTokenTest</span>
+    <span data-ty>[PASS] testConstructorMint() (gas: 10651)</span>
+    <span data-ty>Suite result: ok. 1 passed; 0 failed; 0 skipped; finished in 361.83µs (39.46µs CPU time)</span>
+    <br>
+    <span data-ty>Ran 2 tests for test/Container.t.sol:ContainerTest</span>
+    <span data-ty>[PASS] testContainerFull() (gas: 73204)</span>
+    <span data-ty>[PASS] testInitialUnsatisfied() (gas: 18476)</span>
+    <span data-ty>Suite result: ok. 2 passed; 0 failed; 0 skipped; finished in 422.00µs (128.67µs CPU time)</span>
+    <span data-ty>Ran 2 test suites in 138.17ms (783.83µs CPU time): 3 tests passed, 0 failed, 0 skipped (3 total tests)</span>
+    <span data-ty="input"></span>
```

Comment:
this should also have the file-path, as it is now it just renders:

```
moonbeam@ubuntu-jammy:
```

instead of

```
moonbeam@ubuntu-jammy:~$
```